### ISSUE TITLE
allow command line to reference specific window

### ISSUE THREAD

**Issue by masukomi** [masukomi/fenestro#9]
I'm thinking of something like the following: 
- add an argument that causes fenestro to return a unique identifier for the window your html was loaded into
- if that argument is passed with an id have fenestro _replace_ the contents of the window with that ID 

so, something like `fenestro -p <some_file> -id`    will print a UUID to STDOUT then `fenestro -p <some_file> -id <the uuid>`  would take the contents of the file and load them into the window with that UUID OR (if it was deleted) create a new one and give it that id (for later reuse).

**Comment by kareman** [comment]
So if fenestro is called twice within half a second with the same UUID the contents of the window with that ID should be replaced, as opposed to adding the HTML page to the file list on the left?
@masukomi

**Comment by masukomi** [comment]
yup. @kareman - it would be silly for a user to do that, but that would be the expected behavior.

**Comment by masukomi** [comment]
closing.
Fenestro's been rewritten in Go and has a new home](https://github.com/dacharyc/fenestro).
